Common/OpenCL/ITKimprovements/itkOpenCLLogger.cxx

```diff
@@ -104,10 +104,6 @@ OpenCLLogger::Initialize()
   if (this->m_FileStream->fail())
   {
     itkExceptionMacro(<< "Unable to open file: " << logFileName);
-    delete this->m_FileStream;
-    this->m_FileStream = nullptr;
-    this->m_Created = false;
```

Comment:
@dpshamonin Do you happen to know, should these three lines of code (starting with `delete this->m_FileStream`) be placed _before_, instead of _after_ the `itkExceptionMacro` call?